Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 gem 'rake'
-gem 'cocoapods', '~> 1.10'
+gem 'cocoapods', '~> 1.11'
```

Comment:
This was not _strictly_ necessary, as the most important thing was to ensure the version which was locked in the `Gemfile.lock` lockfile (after running `bundle update cocoapods`) was the same as the one [here](https://github.com/wordpress-mobile/WordPress-iOS/blob/0f033e9e64fc51412b1013026c5da7e0a0e7d7b7/Podfile.lock#L852), and that version `1.11.2` already fulfilled the `~> 1.10` semver requirement. But that doesn't hurt either and made sense to request this as minimum version anyway, so 🙃